modules/serverless-rn-1-32-0.adoc

```diff
@@ -0,0 +1,65 @@
+
+* The secret filtering is enabled by default on the {ServerlessOperatorName} side. An environment variable, `ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID=true`, is added by default to the `net-istio` and `net-kourier` controller pods.
+
+* The `domain-mapping` and `domain-mapping-webhook` deployments functionality in the `knative-serving` namespace is now removed.
+
+* If you have set `spec.config.domain` in the KnativeServing Custom Resource (CR), the default external domain will no longer auto-populate the config-domain ConfigMap in the `knative-serving` namespace. You will now need to configure it manually to ensure accurate domain settings. 
+
+* You can now use the gRPC health probe for `net-kourier` deployments. The `net-kourier-controller` now uses a standard Kubernetes gRPC health probe for both readiness and liveness, replacing its previous use of exec and custom commands. This simplifies the process and aligns it with Kubernetes standards. Additionally, the timeoutSeconds value has been adjusted from 100 milliseconds to 1 second to ensure more reliable probe responses.
+
+* The new trigger filters feature is now available as a Technology Preview. The new trigger filters are now enabled by default. It allows users to specify a set of filter expressions, where each expression evaluates to either true or false for each event.
+
+* Knative Eventing now offers support for data in transit encryption (Eventing TLS) as a developer preview. You can configure Knative Eventing components to encrypt data in-transit by exposing HTTPS addresses and accepting user-provided CA trust bundles. 
+
+* You can now use the Custom Metrics Autoscaler (KEDA) operator to autoscale Knative Eventing sources for Apache Kafka (KafkaSource), both up and down. This functionality is available as a developer preview, offering enhanced scalability and efficiency for Kafka-based event sources within Knative Eventing.
+
+* You can now explore the Knative Eventing monitoring dashboards directly within the *Observe* tab of the developer view in the OpenShift Developer Console. 
+
+* The support for EventTypes `v1beta1` in Knative shipped is deprecated in {ServerlessProductName} 1.32. In {ServerlessProductName} 1.32, the Knative CLI utilizes the EventType v1beta2 API to facilitate the new reference model. The `kn` is not backward compatible with the EventType API v1beta1 in previous releases, specifically limited to the `kn eventtypes` sub-commands group. Therefore, it is recommended to use a matching `kn` version for the best user experience.  
+
+* {ServerlessProductName} now supports custom CA bundle injection for deployments and stateful sets. Additionally, it stores the `SSL_CERT_DIR` environment variable if it was not provided previously. For more information, see link:https://docs.openshift.com/container-platform/latest/networking/configuring-a-custom-pki.html#certificate-injection-using-operators_configuring-a-custom-pki[Configuring a custom PKI].
```

Comment:
```suggestion
* {ServerlessProductName} now supports custom OpenShift CA bundle injection for system components. For more information, see link:https://docs.openshift.com/container-platform/latest/networking/configuring-a-custom-pki.html#certificate-injection-using-operators_configuring-a-custom-pki[Configuring a custom PKI].
```

Reply:
Shouldn't the link to OCP docs be a relative reference vs the current absolute one?

Reply:
@pierDipi We've separated the Serverless docs from the OCP ones, so now we can only do absolute references to OCP docs.